Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Animations.swift

```diff
@@ -40,7 +40,7 @@ extension Effect where Failure == Never {
 
 struct AnimationsState: Equatable {
   var alert: AlertState<AnimationsAction>?
-  var circleCenter = CGPoint(x: UIScreen.main.bounds.width / 2, y: UIScreen.main.bounds.height / 2)
+  var circleCenter = CGPoint(x: 175, y: 300)
```

Comment:
We have to hard code this due to isolation errors in Xcode 14.

Reply:
Since we have a GeometryReader already in place, we can pass the initial center at `onAppear`, isn't it? This is a frequent pattern (initializing values from SwiftUI), so it may be worth presenting an example if this isn't happening in another study.